edb/schema/pointers.py

```diff
@@ -917,54 +1044,66 @@ def _process_create_or_alter_ast(self, schema, astnode, context):
         elif target_ref is not None:
             self._set_pointer_type(schema, astnode, context, target_ref)
 
-    @classmethod
-    def _extract_union_operands(cls, expr, operands):
```

Comment:
I removed this function because it's not used in the whole test suite, and even from the typing point of view it was giving an hard time. Is it correct to remove it?

Reply:
Yeah, seems dead